File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
class Solution {
    public int[] solution(long n) {
        String nToStr = String.valueOf(n);
        int[] answer = new int[nToStr.length()];
        
        int j = 0;
        for (int i = nToStr.length() - 1; i >= 0; i--) {
            answer[j] = Integer.parseInt(Character.toString(nToStr.charAt(i)));
            j++;
        }
        return answer;
    }
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cs
public class Solution {
    public int[] solution(long n) {
        string str = n.ToString();
        int[] answer = new int[str.Length];
        int i = 0;
        while (n > 0) {
            answer[i] = (int)(n % 10);
            n /= 10;
            i++;
        }
        return answer;
    }
}